src/Security/MemberAuthenticator/SessionAuthenticationHandler.php

```diff
@@ -82,11 +82,6 @@ public function logIn(Member $member, $persistent = false, HTTPRequest $request
         if (Member::config()->get('login_marker_cookie')) {
             Cookie::set(Member::config()->get('login_marker_cookie'), 1, 0);
         }
-
-        // Activate sudo mode on login so the user doesn't have to reauthenticate for sudo
-        // actions until the sudo mode timeout expires
-        $service = Injector::inst()->get(SudoModeServiceInterface::class);
```

Comment:
Agreed offline that sudo mode should NOT be automatically activated when you log in